Sources/MongoSwift/ReadConcern.swift

```diff
@@ -2,53 +2,55 @@ import CLibMongoC
 
 /// A struct to represent a MongoDB read concern.
 public struct ReadConcern: Codable {
+    /// Local ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-local/.
+    public static let local = ReadConcern(.local)
+    /// Available ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-available/.
+    public static let available = ReadConcern(.available)
+    /// Linearizable ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-majority/.
+    public static let linearizable = ReadConcern(.linearizable)
+    /// Majority ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-linearizable/.
+    public static let majority = ReadConcern(.majority)
+    /// Snapshot ReadConcern, see https://docs.mongodb.com/master/reference/read-concern-snapshot/.
+    public static let snapshot = ReadConcern(.snapshot)
+    /// Empty ReadConcern.
+    public static let empty = ReadConcern(nil)
+
+    public static func other(_ level: Level) -> ReadConcern {
+        ReadConcern(level)
+    }
+
+    public static func other(_ level: String) -> ReadConcern {
+        ReadConcern(Level(level))
+    }
+
     /// An enumeration of possible ReadConcern levels.
-    public enum Level: RawRepresentable, Codable, Equatable {
+    public struct Level: Codable, Equatable {
         /// See https://docs.mongodb.com/manual/reference/read-concern-local/
-        case local
+        public static let local = Level("local")
         /// See https://docs.mongodb.com/manual/reference/read-concern-available/
-        case available
+        public static let available = Level("available")
         /// See https://docs.mongodb.com/manual/reference/read-concern-majority/
-        case majority
+        public static let majority = Level("majority")
         /// See https://docs.mongodb.com/manual/reference/read-concern-linearizable/
-        case linearizable
+        public static let linearizable = Level("linearizable")
         /// See https://docs.mongodb.com/master/reference/read-concern-snapshot/
-        case snapshot
-        /// Any other read concern level not covered by the other cases.
-        /// This case is present to provide forwards compatibility with any
-        /// future read concerns which may be added to new versions of MongoDB.
-        case other(level: String)
+        public static let snapshot = Level("snapshot")
+
+        public var name: String
```

Comment:
I don't think this needs to be public. If we want to expose this as a `String`, then we can conform the type to [`CustomStringConvertible`](https://developer.apple.com/documentation/swift/customstringconvertible).